modules/installation-special-config-kmod.adoc

```diff
@@ -211,25 +303,39 @@ and makes sure that the `kmods-via-containers@simple-kmod.service` gets started
 on boot:
 ====
 
-. Get a RHEL 8 system, then register and subscribe it:
+. Register a RHEL 8 system:
 +
+[source,terminal]
 ----
 # subscription-manager register
-Username: yourname
-Password: ***************
+----
+
+. Attach a subscription to the RHEL 8 system:
++
+[source,terminal]
+----
 # subscription-manager attach --auto
 ----
 
 . Install software needed to build the software:
 +
+[source,terminal]
 ----
 # yum install podman make git -y
 ----
 
 . Create an Ignition config file that creates a systemd unit file:
+.. Create a directory to host the Ignition config file:
 +
+[source,terminal]
 ----
 $ mkdir kmods; cd kmods
+----
+
+.. Create the Ignition config file that creates a systemd unit file:
++
+[source,terminal]
+----
```

Comment:
Same issue I noted before about `cat <<EOF` examples are input all the way down to the last `EOF`. But the text after the first line all appears grey, so it looks like output.